lib/Condition.ts

```diff
@@ -329,5 +329,6 @@ Condition.prototype.requestObject = function (this: Condition, settings: Conditi
 			return object;
 		}, {[settings.conditionString]: settings.conditionStringType === "array" ? [] : "", "ExpressionAttributeNames": {}, "ExpressionAttributeValues": {}});
 	}
-	return main(this.settings.conditions);
+
+	return utils.object.clearEmpties(main(this.settings.conditions));
```

Comment:
Also. How are we handling this in other cases? We might have to remove some code in other places we call this method to remove the empty objects.